packages/configure-splash-screen/package.json

```diff
@@ -56,11 +56,13 @@
     "@types/fs-extra": "^8.1.0",
     "@types/inquirer": "^6.5.0",
     "@types/jest": "^25.2.1",
+    "@types/sharp": "^0.25.0",
     "babel-jest": "^26.0.1",
     "eslint": "^6.8.0",
     "jest": "^26.0.1",
     "memfs": "^3.1.2",
     "rimraf": "^3.0.2",
+    "sharp": "^0.25.2",
```

Comment:
yeah we should avoid a hard dependency on sharp, it creates too many issues. we use `@expo/image-utils` to handle this elsewhere: https://github.com/expo/expo-cli/tree/master/packages/image-utils - it will use `sharp` if `sharp-cli` is installed, otherwise it falls back to `jump`. if @expo/image-utils doesn't yet expose the functionality you need, then it would be good to add it there (will require a jimp and sharp implementation).